src/js/components/help/helpContent.jsx

```diff
@@ -78,6 +78,14 @@ export default class HelpContent extends React.Component {
                 If you have questions about Broker access, or if your agency is not listed on the MAX page linked above, email <a href='mailto:DATAPMO@fiscal.treasury.gov?subject=Broker%20Access%20Information' target='_top'>DATAPMO@fiscal.treasury.gov</a>.
             </p>
             </div>;
+        }else{
```

Comment:
Generally, I believe our else statements are on a new line with a space before the `{` (I'm already asking for something else, might as well fix this while you're there)